charts/vault-operator/templates/_helpers.tpl

```diff
@@ -34,9 +34,12 @@ Create chart name and version as used by the chart label.
 {{/*
 Overrideable version for container image tags.
 */}}
-{{- define "vault-operator.bank-vaults.version" -}}
+{{- define "vault-operator.vault-operator.version" -}}
 {{- .Values.image.tag | default (printf "%s" .Chart.AppVersion) -}}
```

Comment:
I assume semver of Helm charts (`.Chart.Appversion`) is not equal to `.Values.image.tag`, right? Is this a good default?

Reply:
The default falls back to `.Chart.AppVersion`. Seems to be right